client/src/components/Home.jsx

```diff
@@ -9,62 +9,86 @@ import moment from 'moment';
 // For example data:
 import { result } from '../../../database/example.js';
 
-
   // const naviBar = (<TopBar isMobile={isMobile} onCalendar={onCalendar} setOnCalendar={setOnCalendar}/>)
   // const toDoList = (<ToDoList draggedEvent={draggedEvent} setDraggedEvent={setDraggedEvent} handleDragStart={handleDragStart} myEvents={myEvents}/>)
   // const myCalender = (<MyCalendar myEvents={myEvents} moveEvent={moveEvent} resizeEvent={resizeEvent} changeTitle={changeTitle} onDropFromOutside={onDropFromOutside}/>)
   // const testToDo = (<TestToDo draggedEvent={draggedEvent} setDraggedEvent={setDraggedEvent} handleDragStart={handleDragStart}/>)
   // condition redering base on device
 const Home = ({isMobile, isLoggedIn, isLoading, setIsLoggedIn, userEmail, sharedBy}) => {
+  var exampleData = null;
```

Comment:
I think we might want a comment for this block of code for indicating that it is only for the DEMO part of the app.